tunnel/main.mk

```diff
@@ -2,12 +2,12 @@
 ########################################################################################################################
 SSH_CONFIG ?= $(INFRA_DIR)/env/$(ENV)/ssh.config
 
-CMD_BASTION_SSH_TUNNEL_CONFIG = echo $(shell cat $(INFRA_DIR)/env/$(ENV)/output.json | $(JQ) -r '.ssh_forward_config.value[]' > $(SSH_CONFIG)) && echo "\# SSH Tunnel Config \n\# Use the Forward ports to connect to remote instances (localhost:<PORT>)\n-----" && cat $(SSH_CONFIG)
+CMD_BASTION_SSH_TUNNEL_CONFIG_CREATE = echo $(shell $(AWS) ssm get-parameter --name "/$(ENV)/terraform-output" --with-decryption|$(JQ) -r '.Parameter.Value'|$(BASE64)  -d|$(JQ) -r '.ssh_forward_config.value[]' > $(SSH_CONFIG)) && echo "$(HASHSIGN) SSH Tunnel Config \n$(HASHSIGN) Use the Forward ports to connect to remote instances (localhost:<PORT>)\n-----" && cat $(SSH_CONFIG)
```

Comment:
Spaces:
- Not enough spaces: `--with-decryption|$(JQ) `
- Remove extra spaces `$(BASE64)  -d`